### DISCUSSION TITLE
4/30(일) tableView와 collectionView의 차이점을 알아와주세요

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#117] [General]
tablewView를 사용할 때 좋은 경우와 collectionView를 사용할 때 좋은 경우를 알아보면 좋을 것 같습니다. 둘이 사용하는 것에는 차이점을 알아오면 적절할 때 적절한 코드를 작성할 수 있다고 생각합니다. 😌

**Comment by Eunho0922** [comment]
저번에 내 주셨던 숙제랑은 다른 것 인가요?

**Reply by jjunhaa0211** [reply]
헉 미안 과제를 잘 못냈다 ㅠㅠ 미안...
